content/hardware/07.opta/opta-family/opta/tutorials/02.getting-started-with-rs485/content.md

```diff
@@ -29,10 +29,10 @@ This tutorial will show the steps to connect two Opta™ devices via RS-485 and
 
 ### Required Hardware and Software
 
-- Opta™ PLC with RS-485 support (x2)
-- 12-24VDC/1A power supply (x1)
+- [Opta™ PLC with RS-485 support](https://store.arduino.cc/products/opta-wifi) (x2)
+- 12-24 VDC/1 A power supply (x1)
```

Comment:
I think it would provide better legibility with the following suggestion on top of the change:

```suggestion
- 12-24 VDC / 1 A power supply (x1)
```

What are your thoughts on these suggested changes?

Reply:
not 100% convinced but let's try it